src/features/presentation/pages/presentation/presentation.page.html

```diff
@@ -5,7 +5,10 @@
     <div class="container">
       <div class="row">
         <div class="col-md-6 my-auto p-5">
-          <img src="/assets/img/presentation/personnages_sur_carte.svg" class="img-fluid" alt="" />
+          <img
+            src="/assets/img/presentation/personnages_sur_carte.svg"
+            class="img-fluid"
+            alt="Personnages sur une carte géographique" />
```

Comment:
```suggestion
          <img
            src="/assets/img/presentation/personnages_sur_carte.svg"
            class="img-fluid"
            alt="" />
```
La description est très bien, on peut bien se représenter ce qui existe sur l'image, mais il est important de ne décrire que les images qui apportent une information essentielle au contexte et qui n'est pas déjà accompagné d'un texte descriptif.
Ici, s'aggisant d'une image dont le seul but est d'illustrer la page, il n'y a pas besoin d'ajouter une description.